### PR TITLE
Re-adding versions in Values file

### PR DESCRIPTION
**Pull request by oybed**
I'd like to ensure we stay consistent between the branches (thinking beyond test env), so re-introducing these versions - even though they just point to `master`. Also, I know this won't work for the images as it's using `latest`, but let's chat to figure out proper solution around that.

### CODE REVIEW

File: applications/values.yaml [hunk 1]
```diff
@@ -11,7 +11,10 @@ server: https://kubernetes.default.svc
 ##################
 
 applicationVersions:
+  frontend: &frontendVersion naster
```
Comment (jacobsee) [inline applications/values.yaml:14]: typo 🐱

File: applications/values.yaml [hunk 2]
```diff
@@ -31,6 +37,9 @@ applications:
 - name: omp-backend
   url: https://github.com/rht-labs/open-management-portal-backend.git
   path: deployment
+  ref: *backendVersion
+  helmValues:
+    imageTag: *backendImageTag
```
Comment (jacobsee) [inline applications/values.yaml:42]: should be `backendVersion`

File: applications/values.yaml (hunk 1 above)
Comment (oybed) [reply applications/values.yaml]: Done

File: applications/values.yaml (hunk 2 above)
Comment (oybed) [reply applications/values.yaml]: Done

**Comment by oybed** [comment]
@pabrahamsson yeah, but I'd like to have the version set in only one place, so from that perspective I'm thinking we should just update the 3 repos and use `master`. I'll create the 3 PRs and we can discuss as part of the image v.s. imagestream convo I've mentioned a few times.

**Comment by oybed** [comment]
With the merge of the following PRs, this one should now be good to merge:

https://github.com/rht-labs/open-management-portal-frontend/pull/94
https://github.com/rht-labs/open-management-portal-git-api/pull/50
https://github.com/rht-labs/open-management-portal-backend/pull/56

**Pull request merged**